tests/integration/security/rbac_v1alpha1_test.go

```diff
@@ -277,13 +277,15 @@ func TestV1_Path(t *testing.T) {
 				With(&a, echo.Config{
 					Service:   "a",
 					Namespace: ns,
+					Subsets:   []echo.SubsetConfig{{}},
```

Comment:
Just for education purpose, if we do not set this, does it use the default value of empty array?

Reply:
https://github.com/istio/istio/pull/21369/files#diff-f44759956292d125b633aaa11aeb8544R181
yes